src/components/AddNewBook.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Book.css';
+
+function Book() {
+  return (
+    <>
+      <hr />
+      <h2>ADD NEW BOOK</h2>
+      <form action="">
+        <div className="formcontrol">
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add title"
+            name="title"
+          />
+          <input
+            type="text"
+            className="input-text"
+            placeholder="Add author"
+            name="author"
+          />
+        </div>
+      </form>
```

Comment:
- [ ] Great job implementing the required changes from the previous reviewer. However, there's little thing missing here.
- [ ] Your form has no button to trigger the creation of the book. Kindly look into it and fix.

### Your Design
![your-design](https://github.com/abenezer-addisu/React-Bookstore/assets/26597432/c6a863c6-db85-49bb-99df-abd3e999dd94)